imod/protocols/protocol_applyTransformationMatrix.py

```diff
@@ -110,9 +110,10 @@ def computeAlignmentStep(self, tsObjId):
         # Check if rotation angle is greater than 45º. If so,
         # swap x and y dimensions to adapt output image sizes to
         # the final sample disposition.
-        if rotationAngleAvg > 45 or rotationAngleAvg < -45:
+        if 45 < abs(rotationAngleAvg) < 135:
```

Comment:
Are we sure about this change (asking from my huge ignorance)?

Reply:
yes, this is taken from copytomocoms